nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/QueryDatabaseTable.java

```diff
@@ -120,6 +120,15 @@
             .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor MAX_FRAGMENTS = new PropertyDescriptor.Builder()
+            .name("Maximum Number of Fragments")
+            .description("The maximum number of fragments. If not specified, there is no maximum. " +
+                    "This prevents OutOfMemoryError when this processor ingests huge table.")
+            .defaultValue("1000")
```

Comment:
Since this is a new property and it is optional, the default behavior should be the same as if the property were not there (so the behavior does not change for flows created before the property was available). I'm thinking the default value should be empty.